src/main/java/harmonised/pmmo/config/Config.java

```diff
@@ -72,35 +73,37 @@ private static void setupClient(ForgeConfigSpec.Builder builder) {
 	private static void buildGUI(ForgeConfigSpec.Builder builder) {
 		builder.comment("Configuration settings for the guis").push("GUI");
 		
-		SKILL_LIST_OFFSET_X = builder.comment("how far right from the top left corner the skill list should be")
-						.define("Skill List Xoffset", 0);
-		SKILL_LIST_OFFSET_Y = builder.comment("how far down from the top left corner the skill list should be")
-						.define("Skill List Yoffset", 0);
-		SKILL_LIST_DISPLAY = builder.comment("Should the skill list be displayed")
+		SKILL_LIST_OFFSET_X = builder.comment("How far right from the top left corner the skill list should be (0.0 - 1.0)")
+						.define("Skill List Xoffset", 0.0f);
+		SKILL_LIST_OFFSET_Y = builder.comment("How far down from the top left corner the skill list should be (0.0 - 1.0)")
+						.define("Skill List Yoffset", 0.0f);
+		SKILL_LIST_DISPLAY = builder.comment("Should the skill list be displayed?")
 						.define("Display Skill List", true);
-		VEIN_GAUGE_OFFSET_X = builder.comment("how far right from the bottom left corner the vein guage sholud be")
-				.define("Vein Gauge Xoffset", 4);
-		VEIN_GAUGE_OFFSET_Y = builder.comment("how far up from the bottm left corner the vein guage should be")
-				.define("Vein Gauge Yoffset", 15);
-		VEIN_GAUGE_DISPLAY = builder.comment("Should the vein charge data be displayed")
-				.define("Display Veing Gauge", true);
-		GAIN_LIST_OFFSET_X = builder.comment("how far offset from center the gain list should be")
-				.define("Gain List Xoffset", 0);
-		GAIN_LIST_OFFSET_Y = builder.comment("how far down from the top left corner the gain list should be")
-				.define("Gain List Yoffset", 0);
-		GAIN_LIST_DISPLAY = builder.comment("Should the Gain list be displayed")
+		VEIN_GAUGE_OFFSET_X = builder.comment("how far right from the bottom left corner the vein gauge should be (0.0 - 1.0)")
+				.define("Vein Gauge Xoffset", 0.0f);
+		VEIN_GAUGE_OFFSET_Y = builder.comment("How far up from the bottom left corner the vein gauge should be (0.0 - 1.0)")
+				.define("Vein Gauge Yoffset", 0.0f);
+		VEIN_GAUGE_DISPLAY = builder.comment("Should the vein charge data be displayed?")
+				.define("Display Vein Gauge", true);
+		GAIN_LIST_OFFSET_X = builder.comment("How far offset from left edge the gain list should be (0.0 - 1.0)")
+				.define("Gain List Xoffset", 0.5f);
+		GAIN_LIST_OFFSET_Y = builder.comment("How far down from the top left corner the gain list should be (0.0 - 1.0)")
+				.define("Gain List Yoffset", 0.85f);
+		GAIN_LIST_CENTERED = builder.comment("Should the gain list be centered?")
+				.define("Gain List Centered", true);
+		GAIN_LIST_DISPLAY = builder.comment("Should the Gain list be displayed?")
```

Comment:
Just a bit of advice.  If you use `builder#defineInRange` it automatically adds a comment that includes the min/max range.